client/block-builder/src/lib.rs

```diff
@@ -169,20 +173,22 @@ where
     ///
     /// This will ensure the extrinsic can be validly executed (by executing it).
     pub fn push(&mut self, xt: <Block as BlockT>::Extrinsic) -> Result<(), ApiErrorFor<A, Block>> {
-        info!("Pushing transactions without execution");
-        self.extrinsics.push(xt);
+        let exts = &mut self.extrinsics;
+        exts.push(xt.clone());
```

Comment:
Why we no longer can use `self.extrinsics` here?

Reply:
leftover, fixed now